lib/IHP/static/helpers.js

```diff
@@ -1,10 +1,3 @@
-var ihpLoadEvent = new Event('ihp:load');
```

Comment:
This change is a BC break, not sure whether that's worth it. `ihpLoadEvent` is also used somewhere in the file, so we'd need to remove the usages as well in case we want to delete these variables

Reply:
> This change is a BC break, not sure whether that's worth it

In my tests I saw that `ihp:load` was not working as expected. So on the one hand it's BC break, but on the other - people using it might think it's working. Should we ask on Slack who's using `ihp:load`, see if we get people to answer?

Reply:
I just checked and saw we're ourselves using this in several projects:

<img width="620" alt="image" src="https://github.com/digitallyinduced/ihp/assets/2072185/04760dfb-6aa6-4d9c-baa8-b43430fa946e">

e.g. like this:

<img width="552" alt="image" src="https://github.com/digitallyinduced/ihp/assets/2072185/e91ecec2-151c-4ca9-8bcb-8826b1ccda18">

So let's keep this event here. But I'm fine with removing the outdated documentation on this and replacing it